libmeepgeom/meepgeom.cpp

```diff
@@ -289,38 +250,52 @@ static geom_box gv2box(const meep::volume &v)
   return box;
 }
 
-// TODO rename this to something more descriptive like
-//      is_user_defined_material?
 static bool is_variable(material_type mt)
 {
-  return (mt->which_subclass == material_data::MATERIAL_FUNCTION);
+  return (mt->which_subclass == material_data::MATERIAL_USER);
 }
 static bool is_variable(void* md) { return is_variable((material_type) md); }
 
 static bool is_self(material_type md)
```

Comment:
This shouldn't be called `is_self` any more?

Reply:
Fixed, good point.